projects/packages/sync/composer.json

```diff
@@ -10,7 +10,8 @@
 		"automattic/jetpack-options": "^1.12",
 		"automattic/jetpack-password-checker": "^0.1",
 		"automattic/jetpack-roles": "^1.4",
-		"automattic/jetpack-status": "^1.7"
+		"automattic/jetpack-status": "^1.7",
```

Comment:
I believe this change is a better fit for a separate PR, right?

Reply:
looks like this is an error in the initial setup not sure why 14 is here as 11 uses the version